contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -1686,14 +1686,14 @@ execute_batch(PLtsql_execstate *estate, char *batch, InlineCodeBlockArgs *args,
 	PG_CATCH();
 	{
 		/* Delete temporary tables as ENR */
-		pltsql_remove_current_query_env();
+		pltsql_remove_current_query_env(true);
```

Comment:
Is this guaranteed to be ABORT case?

Reply:
To be honest, I am not 100% sure, but I would assume that any code path within `pltsql_indline_handler()` which throws an error would be trying to abort the transaction.